playwright/test/landing-pages.test.ts

```diff
@@ -16,16 +16,39 @@ test.describe('With JavaScript disabled', () => {
 
   test(`the what's on page displays all events`, async ({ page }) => {
     await gotoWithoutCache(whatsOnUrl, page);
-    const links = page.locator('a:has-text("View all events")');
-    const linksCount = await links.count();
+    const links = await page
+      .getByRole('link', { name: 'View all events' })
+      .all();
+    const linksCount = links.length;
 
     for (let linkIndex = 0; linkIndex < linksCount; linkIndex++) {
-      const link = links.nth(linkIndex);
+      const link = links[linkIndex];
       await expect(link).toBeVisible();
     }
   });
 });
 
+test.describe('With JavaScript enabled', () => {
+  test(`the what's on page displays events by months in switchable tabs`, async ({
+    page,
+  }) => {
+    await gotoWithoutCache(whatsOnUrl, page);
+    const links = await page
+      .getByRole('link', { name: 'View all events' })
+      .all();
+    const linksCount = links.length;
+
+    for (let linkIndex = 0; linkIndex < linksCount; linkIndex++) {
+      const link = links[linkIndex];
+      if (linkIndex === 0) {
+        await expect(link).toBeVisible();
+      } else {
+        await expect(link).toBeHidden();
```

Comment:
I wasn't really sure what this was testing, but I think I get it (there is a "view all events" button for each month but only one is displayed?). Maybe worth changing the description so it matches what it tests a bit more closely?

Reply:
That's the one I thought was duplicated! But it isn't. I did change the description, adding "by months in switchable tabs" once I understood what it does. Could add "only current/active month being visible"?

Reply:
Love that suggestion 👍

Reply:
Done ✅